### DISCUSSION TITLE
Is there an option to show the names of regions/large craters on the Moon?

### DISCUSSION THREAD

**Discussion by issantos** [Stellarium/stellarium#1962] [Q&A] [answered]
Is there an option to show the names of named features on the surface of the Moon within Stellarium? Or perhaps is there an add-on that does this?

**Comment by alex-w** [comment, marked as answer]
See "View" dialog -> Solar System Objects tab ("SSO") -> Planetary nomenclature

**Reply by kishorekumar62** [reply]
Can this flag be toggled from a script?

**Reply by alex-w** [reply]
`NomenclatureMgr.setFlagShowNomenclature(true);`

**Reply by kishorekumar62** [reply]
Thank you for the quick reply!

Where can I find the documentation for NomenclatureMgr? It is not part of the scripting API documentation.

**Reply by alex-w** [reply]
> Where can I find the documentation for NomenclatureMgr? It is not part of the scripting API documentation.

Please see https://stellarium.org/doc/25.0/classNomenclatureMgr.html

**Reply by kishorekumar62** [reply]
Thank you.
